components/Footer/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+import Link from "next/link";
+import React from "react";
+import { Logo } from "../Logo/Logo";
+
+export interface Link {
+  label: string;
+  route: string;
+}
+
+interface FooterLinkProps extends React.HTMLProps<HTMLLIElement> {
+  route: string;
+}
+
+interface FooterProps extends Partial<React.HTMLProps<HTMLElement>> {
+  links: Link[];
+}
+
+const focusClasses = "focus:shadow-md focus:border-white focus:border-[2px]";
+
+export const Footer = ({ links }: FooterProps) => {
+  return (
+    <footer
+      className="flex items-center justify-center bg-blue pb-14 pt-10 md:pt-[52px]"
+      aria-label="Footer"
+      data-testid="footer"
+    >
+      <div className="mx-auto flex w-full max-w-[1040px] flex-col items-center justify-between md:flex-row">
```

Comment:
In this container, we can add some horizontal padding, so that way the contents doesn't go too close to the borders of the screen.

`w-full` is probably not doing anything (please, correct me if I'm wrong).

`max-w-[1040px]` [let's find](https://tailwindcss.com/docs/max-width) the closest to `1040px`